### ISSUE TITLE
pgcleanup complains about disposable_email_whitelist

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy#9780]
Error is:

```
ERROR _process_config(): CONFIGURATION ERROR: Can't open supplied whitelist file from path: /gpfs1/data/galaxy_server/galaxy-dev/config/disposable_email_whitelist.conf
```

**Comment by ic4f** [comment]
@bernt-matthias does that file exist at the specified location? (I assume not?) And if not, is the path wrong, or is there no file at all?

**Comment by bernt-matthias** [comment]
The path to the config dir is correct. The file is not there, but only disposable_email_blacklist.conf.sample

**Comment by ic4f** [comment]
Yep, confirmed, same happens for me. Working on it..

**Comment by ic4f** [comment]
It happens on every galaxy startup, not just in pgcleanup. And why wouldn't it: we are checking if the file path is set, and if so, we try to open the file. Except the value is the default loaded from the schema. I'll fix it, will backport to 20.05.

**Comment by scholtalbers** [comment]
Just a FYI, it looks like this fix was not backported to 20.05.
I worked around it by `touch config/disposable_email_whitelist.conf`
